Makefile

```diff
@@ -13,6 +13,8 @@ help:
 	@echo "make analyze           Slower and more thorough code quality analysis (pylint)"
 	@echo "make format            Correctly format the code"
 	@echo "make checkformatting   Crash if the code isn't correctly formatted"
+	@echo "make isort             Correctly sort and group Python imports"
+	@echo "make checkisort        Crash if Python imports aren't correctly sorted and grouped"
 	@echo "make test              Run the unit tests"
```

Comment:
Could we do this as part of "make format" and "make checkformatting"? This is a formatting step, just a different kind. It also means one less thing to remember to run.